.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```

Comment:
Did you know you can add `--format doc` to your `.rspec` file to get more detailed feedback on your tests? Just an FYI.